corehq/sql_db/connections.py

```diff
@@ -13,7 +14,7 @@
 
 from .util import select_db_for_read
 
-DEFAULT_ENGINE_ID = 'default'
+DEFAULT_ENGINE_ID = DEFAULT_DB_ALIAS
```

Comment:
just for my understanding, is there a reason why these constants are under `corehq/sql_db/connections.py` and not `corehq/sql_db/const.py`

Reply:
there is not `const` module in this app and I don't think it makes sense to move these out of here since they are close to where they are primarily used.

Reply:
i meant in `const.py` under the `sql_db` module. This is a general pattern in other apps so suggested. Fine to ignore.